### PR TITLE
[Console] Don't cut Urls wrapped in SymfonyStyle block

### PR DESCRIPTION
**Pull request by GromNaN**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | https://github.com/symfony/symfony/issues/30920
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/16476

Continuation of https://github.com/symfony/symfony/pull/45318 by @fchris82

### CODE REVIEW

File: src/Symfony/Component/Console/Style/SymfonyStyle.php [hunk 1]
```diff
@@ -450,13 +452,14 @@ private function writeBuffer(string $message, bool $newLine, int $type): void
 
     private function createBlock(iterable $messages, string $type = null, string $style = null, string $prefix = ' ', bool $padding = false, bool $escape = false): array
     {
+        $this->outputWrapper ??= new OutputWrapper();
```
Comment (fabpot) [inline src/Symfony/Component/Console/Style/SymfonyStyle.php]: Seems like a micro-optimization. What about creating a new instance every time we call this method?

**Review by fabpot — APPROVED** [review]
LGTM, except for a small suggestion.

**Comment by fabpot** [comment]
Looks like tests are broken by this PR.

**Comment by fabpot** [comment]
Thank you @GromNaN.

**Comment by GromNaN** [comment]
Thank you @fchris82

**Pull request merged**